fineract-provider/src/main/resources/db/changelog/tenant/commands/insert_portfolio_management_table_reports.sql

```diff
@@ -0,0 +1,136 @@
+-- changeset fineract:1
+-- MySQL dump 10.13  Distrib 5.1.60, for Win32 (ia32)
+--
+-- Host: localhost    Database: fineract_default
+-- ------------------------------------------------------
+-- Server version	5.1.60-community
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES UTF8MB4 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+INSERT INTO stretchy_report (report_name, report_type, report_category, report_sql, description, core_report, use_report, self_service_user_report)
+VALUES ("Portfolio Management", "Table", "Loan", "select concat(u.firstname, ' ', u.lastname) as 'User', concat(c.firstname, ' ', c.lastname) as 'Client Name', c.external_id as 'Client UUID', l.account_no as 'Loan Number', loanPurposeTble.code_value as 'Purpose',
+cvd.code_value as 'Department', cvs.code_value as  'Strata', '-' as 'KIVA Loan ID', c.id as 'Client ID', c.date_of_birth as 'Date of Birth', '-' as 'KIVA Client ID', l.loan_counter as 'Cycle', cvc.code_value as 'Cohort',
+'-' as 'Is Startup', cvg.code_value as 'Gender', cvp.code_value as 'Province', cvb.code_value as 'Sector', cvn.code_value as 'Nationality', coi.telephone_no as 'Telephone', l.approvedon_date as 'Approval Date',
+l.disbursedon_date as 'Disbursement Date', l.principal_amount_proposed as 'Applied Amount', l.approved_principal as 'Approved Amount', l.principal_disbursed_derived as 'Disbursed Amount',
+(l.principal_disbursed_derived - l.principal_amount_proposed) as 'Difference', currency.name as 'Currency Type', l.number_of_repayments as 'Re-payment Term',
+        case
+            when l.loan_type_enum = 1 then 'Individual'
+            when l.loan_type_enum = 2 then 'Group'
+            when l.loan_type_enum = 3 then 'JLG'
+            when l.loan_type_enum = 4 then 'GLIM'
+            when l.loan_type_enum = 5 then 'GSIM'
+        end as 'Loan Type',
+l.term_frequency as 'Terms Duration', l.total_repayment_derived as 'Actual Payment Amount(KES)', l.principal_repaid_derived as 'Principal Paid(KES)', l.interest_repaid_derived as 'Interest Paid(KES)',
+l.fee_charges_repaid_derived as 'Fees Paid(kes)', l.penalty_charges_repaid_derived as 'Total Late Fees Paid(KES)', l.total_overpaid_derived as 'Excess Amount Paid(KES)', l.total_outstanding_derived as 'Current Balance(KES)',
+l.principal_outstanding_derived as 'Principal Balance(KES)', l.interest_outstanding_derived as 'Interest Balance(KES)', l.fee_charges_outstanding_derived as 'Fees Balance(KES)', laa.total_overdue_derived as 'Amount Past Due(KES)',
```

Comment:
@mkalema  Is this report  supporting only one currency (KES) ?

Reply:
The excel i got these columns from has these columns as they appear here. I referenced that

Reply:
The excel i got these columns from has these columns as they appear here. I referenced that. Temi says we can remove that and add currency as a parameter